remoteappmanager/db/orm.py

```diff
@@ -129,6 +130,54 @@ def reset(self):
         Base.metadata.create_all(self.engine)
 
 
+class AccountingInterface(ABCAccounting):
+
+    def __init__(self, url, **kwargs):
+        self.db = Database(url, **kwargs)
+
+        # We keep the same session for all transactions
+        # so that the session remains on one thread
+        self.session = self.db.create_session()
```

Comment:
We could augment the ABCAccounting API with `setup`, and `cleanup` methods if we find that we need to manage resources